q2_phylogeny/tests/test_raxml.py

```diff
@@ -25,16 +26,22 @@ class RaxmlTests(TestPluginBase):
 
     package = 'q2_phylogeny.tests'
 
+    def setUp(self):
+        super().setUp()
+        src = Path(__file__).parent / "data"
+        self.data_dir = Path(self.temp_dir.name) / "data"
+        shutil.copytree(src, self.data_dir)
+
+    def get_data_path(self, filename):
```

Comment:
Probably worth putting a comment here letting future readers know this is an overridden method.